connectors/src/resources/zendesk_resources.ts

```diff
@@ -102,6 +106,17 @@ export class ZendeskBrandResource extends BaseResource<ZendeskBrand> {
     return new Ok(undefined);
   }
 
+  /**
+   * Deletes the brand and all associated data (tickets, categories, articles).
+   */
+  // eslint-disable-next-line no-empty-pattern
+  async remove({}: {
```

Comment:
I don't understand the logic to have a `delete` that deletes the brand only, and a `remove` that deletes branch + all children. Could you explain? 🙏🏻

Reply:
More practically, the signature of `delete` is fixed and I needed another method.

Reply:
I can rename `remove` into `deleteWithChildren`, it makes it clear that `delete` only affects the brand and not its children

Reply:
Yes that makes sense. And what's the use case to delete the brand and not the children?

Reply:
Delete brand + all children is performed when the brand is deleted on zendesk, or when permissions are revoked, we never have to only delete the brand but this is an abstract method inherited from the BaseResource and whose signature does not allow passing the arguments I need to delete the children aswell

Reply:
I think I'll deal with this when I will actually implement it, I'm not really sure at this point that this should be a method in the `ZendeskBrandResource`, it could actually turn out to be an activity, to be continued

Reply:
https://github.com/dust-tt/dust/pull/8305